src/mixs/schema/mixs.yaml

```diff
@@ -11339,7 +11364,12 @@ slots:
       - value: 4th floor
     keywords:
       - floor
-    string_serialization: '[1st floor|2nd floor|{integer} floor|basement|lobby]'
+    # \d*1[1-3]th –> 11th, 12th, 13th, 111th, etc.
+    # \d*1st –> 1st, 21st, 31st, 101st, etc.
+    # \d*2nd –> 2nd, 22nd, 32nd, 102nd, etc.
+    # \d*3rd –> 3rd, 23rd, 33rd, 103rd, etc.
+    # \d*[04-9]th –> 4th, 5th, 6th, 10th, 11th, 14th, 20th, etc.
+    pattern: ^(?:(?:\d*1[1-3]th|\d*1st|\d*2nd|\d*3rd|\d*[04-9]th) floor|basement|lobby)$
```

Comment:
This pattern form is better than its predecessor, and makes more sense than a huge enumeration, however:

@sujaypatil96 @turbomam can we open an issue to settle on the dialect of regexes we use across GSC resources?

Reply:
Also, couldn't this be in a `syntax` property of a structured pattern pointing to something like `{FLOOR}` ? Similar to what you do for DOIs etc? Just for consistency.

Reply:
Do you mean you want to represent the entire regex here – `^(?:(?:\d*1[1-3]th|\d*1st|\d*2nd|\d*3rd|\d*[04-9]th) floor|basement|lobby)$` as a `structured_pattern`? Our reasoning/motivation for representing something (a regex) as a structured pattern is if there are use cases where it can be reused multiple times to express a pattern.

In this case, the `samp_floor` term is the only term that has the necessity for this pattern. Also, like @turbomam said on call today, it wasn't economical to represent it as a structured pattern because we didn't just want to do something like `{integer} floor` bur rather express it fully like `1st floor`, `2nd floor`, `3rd floor` and so forth.